src/foam/demos/u2/DetailView.html

```diff
@@ -1,6 +1,6 @@
 <html>
   <head>
-    <script language="javascript" src="../../../foam.js" project="foam3/src/pom"></script>
+    <script language="javascript" src="../../../foam.js" project="pom"></script>
```

Comment:
Is this needed? Can you just remove the project attribute now?

Reply:
It is not needed